modules/ROOT/pages/8.1.0-release-notes.adoc

```diff
@@ -55,6 +55,21 @@ The following premium plugin updates were released alongside {productname} {rele
 // For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].
 
 
+=== Suggested Edits
+
+The {productname} {release-version} release includes an accompanying release of the **Suggested Edits** premium plugin.
+
+**Suggested Edits** includes the following fix:
+
+=== Empty editor operations sometimes showed as modified instead of added/removed
+// #TINY-12273
+
+Previously empty editor operations were sometimes incorrectly displayed as modifications instead of insertions or removals. This behavior caused confusion, as an editor that appeared empty could still show a “modified” state.
+
+To resolve this, empty content is no longer sent. When the editor is empty or becomes empty, the entire state is now correctly represented as inserted or removed. This improvement ensures that change tracking is clearer and more accurate, reducing confusion when editors contain no content.
```

Comment:
```suggestion
Previously empty editor operations were sometimes incorrectly displayed as modifications instead of additions or removals. This behavior caused confusion, as an editor that appeared empty could still show a “modified” state.

To resolve this, empty content is no longer sent. When the editor is empty or becomes empty, the entire state is now correctly represented as added or removed. This improvement ensures that change tracking is clearer and more accurate, reducing confusion when editors contain no content.
```

In the plugin, we use "added/deleted" content. In the Suggested Edits plugin page, we mention "added/removed". Not sure if that matters, just thought of aligning to what we use in the Suggested Edits plugin page.